### PR TITLE
Add preperint typo

### PR DESCRIPTION
**Pull request by yarikoptic**
Detected in neurolibre project: https://github.com/neurolibre/neurolibre/issues/47 in a few spots, so seems to be common, and has a few more hits throughout the github.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -32382,6 +32382,8 @@ prepere->prepare
 prepered->prepared
 preperes->prepares
 prepering->preparing
+preperint->preprint
+preperints->preprints
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:32386]: Perhaps add all verbal forms?
```
preperinted->preprinted
preperinting->preprinting
```

Comment (yarikoptic) [reply codespell_lib/data/dictionary.txt:32386]: why not?
```suggestion
preperints->preprints
preperinted->preprinted
preperinting->preprinting
```

Comment (yarikoptic) [reply codespell_lib/data/dictionary.txt:32386]: ha -- I hit some github bug which forbids me to commit this suggestion... try or I will try later

Comment (DimitriPapadopoulos) [reply codespell_lib/data/dictionary.txt:32386]: Actually, verbals forms are very uncommon: couldn't find the typo in GitHub or with Google.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**